city_scrapers/spiders/chi_ssa_23.py

```diff
@@ -0,0 +1,143 @@
+import logging
+import re
+import unicodedata
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa23Spider(CityScrapersSpider):
+    name = "chi_ssa_23"
+    agency = "Chicago Special Service Area #23 Clark Street"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.lincolnparkchamber.com/clark-street-ssa-administration/"]
+    location = {
+        "name": "Lincoln Park Chamber of Commerce",
+        "address": "2468 N. Lincoln Chicago, IL 60614"
+    }
+    # Each meeting takes place on Wednesday 4 PM
+    meeting_day = 'Wednesday'
```

Comment:
It might make sense to parse this from the page manually, or check if it's changed and raise an error if it has like we've done with `_validate_location` in a couple places like this: https://github.com/City-Bureau/city-scrapers/blob/d7d1fbbe1236c4353be03e289580e1421465c5e8/city_scrapers/spiders/chi_lsc_advisory.py#L80-L83

Reply:
I added this manual check, makes sense.